models/biocro/R/run.biocro.R

```diff
@@ -142,6 +117,7 @@ run.biocro <- function(lat, lon, metfile, soil.nc = NULL, config = config, coppi
                            photoControl = l2n(config$pft$photoParms))
       
     }
+    # TODO return the whole BioGro result instead of selected columns?
```

Comment:
It is easy enough to add columns here ... if you return all of the results the files can increase in size a few fold, which can get quite large (especially for ensembles and regional / hourly output). Feel free to add any additional variables you think would be of interest.